Makefile

```diff
@@ -1,14 +1,19 @@
+install: package.json
+	npm install
+
+lint: install
+	npm run lint
+
 test: lint
 	npm run test
 	npm run coverage
 
-lint: node_modules
-	npm run lint
+publish: test
```

Comment:
You might want to check out our recommendations for releasing here https://github.com/bucharest-gold/entente/blob/master/GUIDELINES.md#release-process